spec/services/processors/jira_project_defect_escape_rate_updater_spec.rb

```diff
@@ -60,6 +57,38 @@
       end
     end
 
+    context 'when the bug is changed to In Progress' do
+      let(:bugs) do
+        [
+          {
+            'key': 'TES-4',
+            'fields': {
+              'customfield_10000': [{ 'value': 'production' }],
+              'created': '2021-03-14T15:48:00.000-0300',
+              'resolutiondate': '2021-03-19T17:30:04.000-0300'
+            },
+            'changelog': {
+              'histories': [
+                {
+                  'created': '2021-03-17T15:48:04.000-0300',
+                  'items': [
+                    'fieldId': 'status',
+                    'fromString': 'To Do',
+                    'toString': 'In Progress'
+                  ]
+                }
+              ]
+            }
+          }
+        ]
+      end
+
+      it 'updates the in progress date' do
+        subject
+        expect(last_issue.in_progress_at).to eq('2021-03-17T15:48:04.000-0300')
```

Comment:
Sometimes I find nice for visibility improvement in the expectations to use the [change matchers](https://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers:change) so you can write things like 
```
expect {
  subject
}.to change(last_issue, :in_progress_at).from(some_initial_date).to(some_expected_date)
```
also move this long string dates to variables defined with _let_